src/main/scala/wdl4s/exception/LookupException.scala

```diff
@@ -1,14 +1,23 @@
 package wdl4s.exception
 
 import lenthall.exception.ThrowableAggregation
-import wdl4s.GraphNode
+import wdl4s.{Declaration, GraphNode}
+import wdl4s.types.WdlType
 
 sealed trait LookupException { this: Exception => }
 
+final case class OptionalNotSuppliedException(operationName: String) extends Exception(s"Sorry! Operation $operationName is not supported on empty optional values. You might resolve this using select_first([optional, default]) to guarantee that you have a filled value.")
+
 /**
   * When a variable does not reference any known scope in the namespace.
   */
-final case class VariableNotFoundException(variable: String) extends Exception(s"Variable '$variable' not found") with LookupException
+sealed abstract case class VariableNotFoundException(variable: String, quoteName: Boolean = true) extends Exception(s"Variable $variable not found") with LookupException
```

Comment:
Follows your blog post's example, it forces library users to use the object's `apply` rather than the default case class `apply`.